docs/unified-test-documentation/dasharo-compatibility/361-esp-scanning.md

```diff
@@ -0,0 +1,166 @@
+# Dasharo Compatibility: ESP Partition Scanning
+
+## Test cases common documentation
+
+**Test setup**
+
+1. Proceed with the
+   [Generic test setup: firmware](../generic-test-setup.md#firmware).
+
+The purpose of this test is to verify that the EFI partition is scanned for
+any OS-specific folders and .efi files, and that corresponding boot menu
+entries are created for them. The list of supported .efi files and their boot
+menu entries is as follows:
+
+* \EFI\Microsoft\Boot\bootmgfw.efi - Windows Boot Manager (on <disk_name>)
```

Comment:
```suggestion
* \EFI\Microsoft\Boot\bootmgfw.efi - Windows Boot Manager (on `<disk_name>`)
```

Such marks have to be in ticks otherwise it will be treated as HTML.

Also this is not exhaustive list. 


* \EFI\qubes\shimx64.efi - Qubes OS (on <disk_name>)
* \EFI\qubes\grubx64.efi - Qubes OS (on <disk_name>)
* \EFI\DTS\grubx64.efi" - Dasharo Tools Suite (on <disk_name>)

DTS option is populated regardless of removable media state

Reply:
Resolved in [recent commit](https://github.com/Dasharo/docs/commit/0d02cb8b01714d8238b369c37f02848b1a991a15)